src/config.rs

```diff
@@ -170,6 +180,16 @@ impl Config {
                 if let Ok(env_token) = env::var(DELETE_TOKEN_ENV) {
                     tokens.insert(env_token);
                 }
+
+                if let Ok(env_path) = env::var(DELETE_FILE_ENV) {
+                    match read_to_string(&env_path) {
+                        Ok(s) => push_token_file(s, &mut tokens),
```

Comment:
No need for a function call:

```suggestion
                        Ok(s) => {
                            s.lines().filter(|l| !l.trim().is_empty()).for_each(|l| {
                                tokens.insert(l.to_string());
                            });
                        }
```

Reply:
I originally created a function to be able to write a UT for the parsing, is just a fixture fine then ?

Reply:
Yup, it should be enough.